feat: implements owner and repositories input for token generation (#277)
camunda/infra-global-github-actions: generate-github-app-token-from-vault-secrets/README.md

| vault-auth-secret-id                | The Secret Id for (Vault) App Role authentication                       |
| vault-url                           | The URL for the Vault endpoint                                          |
| skip-token-revoke                   | If truthy, the token will not be revoked when the current job is complete (optional) |

> | owner                               | The owner of the GitHub App installation (defaults to current repository owner, optional). To leave this value empty in the token generation, set it to `!null`. |

Reviewer: why not just empty string "" instead? I think it's more intuitive like that.

Reviewer: I'd also argue that there's the need to have the case of generating a token for all repositories - normally it's either the current one, or a different (specific) one, wdyt?

Reviewer: basically, wouldn't it be enough to have a thin wrapper for the https://github.com/actions/create-github-app-token and just to pass `owner` and `repositories` to it as is?

Reviewer: Thank you, I didn't think of the Renovate use case, it makes total sense.

> | repositories                        | Comma or newline-separated list of repositories to install the GitHub App on (defaults to current repository if owner is unset, optional). If you want to generate a token that has access to all repositories of the owner, set this to `!all` and explicitely set an `owner`. |

Reviewer: since the app is not installed by this action, only a token obtained:
```suggestion
| repositories                        | Comma or newline-separated list of repositories for which the GitHub app token will be valid for(defaults to current repository if owner is unset, optional). If you want to generate a token that has access to all repositories of the owner, set this to `!all` and explicitely set an `owner`. |
```

> | repositories                        | Comma or newline-separated list of repositories to install the GitHub App on (defaults to current repository if owner is unset, optional). If you want to generate a token that has access to all repositories of the owner, set this to `!null`. |

Reviewer: same as above
